### PR TITLE
Switch from Rocky to alma linux

### PR DESCRIPTION
**Pull request by bhargavdas**

### CODE REVIEW

**Review by chombourger — APPROVED** [review]

**Comment by chombourger** [comment]
changes look good but we could have made a single commit. do you mind squashing them? I have started a CI check to see if our issue is going away. thanks

**Comment by bhargavdas** [comment]
> changes look good but we could have made a single commit. do you mind squashing them? I have started a CI check to see if our issue is going away. thanks

Sure

**Comment by bhargavdas** [comment]
> > changes look good but we could have made a single commit. do you mind squashing them? I have started a CI check to see if our issue is going away. thanks
> 
> Sure

Squashed, Please check if the commit subject is appropriate.

**Review by chombourger — APPROVED** [review]

**Comment by chombourger** [comment]
> > > changes look good but we could have made a single commit. do you mind squashing them? I have started a CI check to see if our issue is going away. thanks
> > 
> > 
> > Sure
> 
> Squashed, Please check if the commit subject is appropriate.

looks good to me and CI checks appear to go further (and are still running)

File: tests/test_storage.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def test_write_gz(powered_off):
 
 def test_write_xz(powered_off):
     assert Storage.to_host() is True
-    Storage.write("rockylinux.tar.xz")
+    Storage.write("almalinux.tar.xz")
```
Comment (chombourger) [inline tests/test_storage.py:54]: the line "assert Console.wait_for("Rocky Linux") is not None" should also be updated

Comment (chombourger) [reply tests/test_storage.py:54]: this will fix the CI failure we just got

Comment (bhargavdas) [reply tests/test_storage.py:54]: Checking the almalinux what the name should be. Assuming it should be "Alma Linux"

Comment (bhargavdas) [reply tests/test_storage.py:54]: Updated the name, it is "AlmaLinux"

**Review by chombourger — APPROVED** [review]

**Pull request merged**